index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/fc7e7ce012.js" crossorigin="anonymous"></script>
+</head>
+<body>
+    <div class="container">
+       <div class="title-container">
+        <h1>Today's To Do</h1>
+        <button class="refresh-btn"><i class="fa-solid fa-rotate fa-2x"></i></button>
+       </div>
+        <hr>
+        <form class="to-do-form" action="">
+            <input id="toDoInput" type="text" placeholder="Add to your list...">
+            <button type="submit"><i class="fa-solid fa-arrow-left fa-2x"></i></button>
+        </form>
+        <hr>
+        <div class="to-do-container"></div>
+        <div class="clear-div">
+            <button class="clear-btn">Clear All Completed</button>
+        </div>
+    </div>
+    <script src="index.js"></script>
```

Comment:
Please Use Defer Attribute in your script tag; it will ensure that the Script 
is Executed properly After the Page Reload.